tests/test_plugins/test_manifest_plugins/test_manifest_plugins.py

```diff
@@ -73,3 +76,21 @@ def test_exported_tags(vulnerable_manifest_path):
     for issue in plugin.issues:
         assert Severity.WARNING == issue.severity
         assert "Manifest" == issue.category
+
+
+@pytest.mark.parametrize("apk_constants", [
+    None,
+    {"min_sdk": 8},
+])
+def test_min_sdk(apk_constants):
+    plugin = MinSDK()
+    plugin.run([os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                             "test_min_sdk_tapjacking",
+                             "androidmanifest.xml")],
+               apk_constants=apk_constants)
```

Comment:
Does this test if apk_constants wasn't generated properly?

Reply:
It tests with apk_constants=None and apk_constants as a proper dictionary.

Reply:
You could also throw in an empty dictionary for this test. Potentially also a dict that doesn't contain a `"min_sdk"` even though there's nothing in the code currently that would use the falsey property of an empty dictionary.